From 83d02632edd6b55a2892c8c5bcfdd01ee636435f Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Mon, 10 Mar 2025 14:48:35 +0100
Subject: [PATCH] drm/bridge: synopsys: dw-mipi-dsi: replace dev_err by dev_dbg

Do not print an error message when a write fails.
This may be normal behavior and a new write sequence
should be started again.

Change-Id: Ib62b268d380e3a36c012dba581d529efda15fc42
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/443168
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/gpu/drm/bridge/synopsys/dw-mipi-dsi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/bridge/synopsys/dw-mipi-dsi.c
+++ b/drivers/gpu/drm/bridge/synopsys/dw-mipi-dsi.c
@@ -432,7 +432,7 @@ static int dw_mipi_dsi_gen_pkt_hdr_write
 				 val, !(val & GEN_CMD_FULL), 1000,
 				 CMD_PKT_STATUS_TIMEOUT_US);
 	if (ret) {
-		dev_err(dsi->dev, "failed to get available command FIFO\n");
+		dev_dbg(dsi->dev, "failed to get available command FIFO\n");
 		return ret;
 	}
 
@@ -443,7 +443,7 @@ static int dw_mipi_dsi_gen_pkt_hdr_write
 				 val, (val & mask) == mask,
 				 1000, CMD_PKT_STATUS_TIMEOUT_US);
 	if (ret) {
-		dev_err(dsi->dev, "failed to write command FIFO\n");
+		dev_dbg(dsi->dev, "failed to write command FIFO\n");
 		return ret;
 	}
 
